lib/ts/recipe/passwordless/components/features/linkClickedScreen/index.tsx

```diff
@@ -109,13 +115,19 @@ const LinkClickedScreen: React.FC<PropType> = (props) => {
     const handleConsumeError = useCallback(
         (err) => {
             if (STGeneralError.isThisError(err)) {
-                return SuperTokens.getInstanceOrThrow().redirectToAuthWithoutRedirectToPath(undefined, props.history, {
-                    error: "custom",
-                    message: err.message,
+                return SuperTokens.getInstanceOrThrow().redirectToAuth({
+                    history: props.history,
+                    queryParams: {
+                        error: "custom",
+                        message: err.message,
+                    },
                 });
             } else {
-                return SuperTokens.getInstanceOrThrow().redirectToAuthWithoutRedirectToPath(undefined, props.history, {
-                    error: "restart_link",
+                return SuperTokens.getInstanceOrThrow().redirectToAuth({
+                    history: props.history,
+                    queryParams: {
+                        error: "restart_link",
+                    },
```

Comment:
need to explicitly set redirectBack to false -> internal use of this function should always explicitly set this value to prevent future bugs.